21-nats-overview/script.md

```diff
@@ -0,0 +1,462 @@
+# Introduction
+
+Hey everyone, my name is Nate and I like NATS! I'm a software architect who's
+been building web applications and infrastructure for about 20 years. And I got a new job!
+
+I recently joined Synadia as a DX engineer, where I'm excited to work on our web
```

Comment:
I'd consider putting this intro at the end instead of the beginning, like we went over in our call